cmd/nodecmd/status.go

```diff
@@ -55,32 +57,43 @@ func statusSubnet(_ *cobra.Command, args []string) error {
 			return ErrNoBlockchainID
 		}
 		notSyncedNodes := []string{}
+		subnetSyncedNodes := []string{}
+		subnetValidatingNodes := []string{}
 		for _, host := range ansibleHostIDs {
-			isSubnetSynced, err := getNodeSubnetSyncStatus(blockchainID.String(), clusterName, host, true)
+			subnetSyncStatus, err := getNodeSubnetSyncStatus(blockchainID.String(), clusterName, host)
 			if err != nil {
 				return err
 			}
-			if !isSubnetSynced {
+			switch subnetSyncStatus {
+			case status.Syncing.String():
+				subnetSyncedNodes = append(subnetSyncedNodes, host)
+			case status.Validating.String():
+				subnetValidatingNodes = append(subnetValidatingNodes, host)
+			default:
 				notSyncedNodes = append(notSyncedNodes, host)
 			}
 		}
-		printOutput(ansibleHostIDs, notSyncedNodes, clusterName, subnetName)
+		printOutput(ansibleHostIDs, notSyncedNodes, subnetSyncedNodes, subnetValidatingNodes, clusterName, subnetName)
 		return nil
 	}
 	notBootstrappedNodes, err := checkClusterIsBootstrapped(clusterName)
 	if err != nil {
 		return err
 	}
-	printOutput(ansibleHostIDs, notBootstrappedNodes, clusterName, subnetName)
+	printOutput(ansibleHostIDs, notBootstrappedNodes, nil, nil, clusterName, subnetName)
 	return nil
 }
 
-func printOutput(hostAliases, notBootstrappedHosts []string, clusterName, subnetName string) {
+func printOutput(hostAliases, notBootstrappedHosts, subnetSyncedHosts, subnetValidatingHosts []string, clusterName, subnetName string) {
 	if len(notBootstrappedHosts) == 0 {
 		if subnetName == "" {
 			ux.Logger.PrintToUser(fmt.Sprintf("All nodes in cluster %s are bootstrapped to Primary Network!", clusterName))
 		} else {
-			ux.Logger.PrintToUser(fmt.Sprintf("All nodes in cluster %s are synced to Subnet %s", clusterName, subnetName))
+			status := "synced"
+			if len(subnetSyncedHosts) == 0 && len(subnetValidatingHosts) > 0 {
+				status = "validators"
+			}
+			ux.Logger.PrintToUser(fmt.Sprintf("All nodes in cluster %s are %s to Subnet %s", clusterName, status, subnetName))
```

Comment:
Sprintf not needed. Also remove "to " to be consistent with previous two comments